x-pack/legacy/plugins/ml/public/services/ml_api_service/data_frame_analytics.js

```diff
@@ -40,6 +40,13 @@ export const dataFrameAnalytics = {
       data: analyticsConfig
     });
   },
+  evaluateDataFrameAnalyticsRegression(evaluateConfig) {
```

Comment:
Do we need `Regression` in the name here? Maybe it will also be used for other future analytics tasks like classification.

Reply:
We do not - good point for re-usability. Updated in 6de9d0d162db3f4225bd32b347bdd502f4d51719